test/integration/pack/test_pack.py

```diff
@@ -156,6 +156,49 @@ def prepare_tgz_test_cases(tt_cmd) -> list:
             ],
             "artifacts_in_separated_dir": False,
         },
+        {
+            "bundle_src": "bundle8",
+            "cmd": tt_cmd,
+            "pack_type": "tgz",
+            "args": ["--cartridge-compat"],
+            "res_file": "app_name-0.1.0.0." + get_arch() + ".tar.gz",
+            "check_exist": [
```

Comment:
Need to check that no other files/dirs exist in archive root. Only app name dir.